test/escape-case.js

```diff
@@ -0,0 +1,111 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/escape-case';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const errors = [{
+	ruleId: 'escape-case',
+	message: 'Use uppercase characters for the value of the escape sequence.'
+}];
+
+ruleTester.run('escape-case', rule, {
+	valid: [
+		'const foo = \'\\xA9\';',
```

Comment:
@jfmengels Thank you for your feedback so far.

I agree with you that double quotes are easier to read but in a previous comment sindresorhus, if I understood him correctly, requested single quotes--so I'm not sure which one to go with :P.

Any thoughts?